### DISCUSSION TITLE
plays/bobby-coder/dad-jokes

### DISCUSSION THREAD

**Discussion by giscus** [reactplay/react-play#956] [Play Feedback]
# plays/bobby-coder/dad-jokes

A web app where you can enjoy the funny Dad Jokes.

https://reactplay.io/plays/bobby-coder/dad-jokes

**Comment by Adityakadali** [comment]
Nice UI. Loved the jokes :rocket: :rocket: :rocket:

**Reply by Bobby-coder** [reply]
Thank you Aditya
